frontend/src/pages/PropertyCreation.jsx

```
@@ -0,0 +1,710 @@
+          <div className="mt-4 grid grid-cols-3 gap-4">
+            {preview.map((image, index) => (
+              <div key={index} className="relative group">
+                <img
+                  src={image.preview}
+                  alt={`preview-${index}`}
+                  className="w-full h-24 object-cover rounded-md"
+                />
+                <button
+                  onClick={(e) => removeImage(index,e)}
+                  className="absolute top-1 right-1 bg-red-600 text-white p-1 rounded-full opacity-80 hover:opacity-100 transition"
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)